firmware/library/L1_Drivers/can.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "L0_LowLevel/LPC40xx.h"
+#include "L2_Utilities/macros.hpp"
+
+SJ2_PACKED(struct) CanMessage_t
+{
+    union
+    {
+        uint32_t frame;
```

Comment:
Can you add comments for each section of this struct with regards to how it relates to how it is going to be used, or its relationship to a can frame.

Reply:
Yes. Will add in next commit.

Reply:
comments added